### PR TITLE
24.1 DS configuration docs updates 

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Update DS config docs to a single-ingress model for easier deployment during 24.1 upgrade cycle.

### CODE REVIEW

File: platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx [hunk 1]
```diff
@@ -122,20 +125,26 @@ This procedure describes how to configure Data Studios for Kubernetes deployment
...
-    - `alb.ingress.kubernetes.io/load-balancer-attributes` annotation:
-    - `spec.rules.host` field:
-
 1. Edit the `server.yml` file and set the `CONNECT_REDIS_ADDRESS` environment variable to the hostname or IP address of the Redis server configured for Platform.
 
 1. Edit the `proxy.yml` file and set the following variables:
 
     - `CONNECT_REDIS_ADDRESS`: The hostname or IP address of the Redis server configured for Platform.
-    - `CONNECT_PROXY_URL`: A base domain name for the connect proxy subdomain. For example, if you specify the base domain `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.
+    - `CONNECT_PROXY_URL`: A base domain URL for the connect proxy subdomain. We recommend you set the base URL for your Platform installation (`PLATFORM_URL` below) for your connect proxy, to be able to use the same wildcard TLS certificate for all data studio instance URLs and avoid additional domain nesting.
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx:133]:
```suggestion
    - `CONNECT_PROXY_URL`: A URL for the connect proxy subdomain. We recommend you set a first-level subdomain of your Platform installation domain (`PLATFORM_URL` below) for your connect proxy, to be able to use the same wildcard TLS certificate for all data studio instance URLs and avoid additional domain nesting. For example, `https://connect.example.com`.
```

File: platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx [hunk 2]
```diff
@@ -68,19 +72,19 @@ This guide assumes that all services will be run in the same container as the re
...
-   1. Set the following environment variables:
-      - `PLATFORM_URL`: The same value assigned to `TOWER_SERVER_URL`.
-      - `CONNECT_PROXY_URL`: A base domain name for the connect proxy subdomain. For example, if you specify the base domain `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.
-      - `CONNECT_OIDC_CLIENT_REGISTRATION_TOKEN`: The same base64-encoded value set in the `client_token` environment variable.
 
-1. Open `data-studios.env` in an editor and set the following variables:
+    1. Uncomment the `connect-proxy` and `connect-server` services.
+    1. Set the following environment variables:
+        - `PLATFORM_URL`: The same value assigned to `TOWER_SERVER_URL`.
+        - `CONNECT_PROXY_URL`: A base domain name for the connect proxy subdomain. For example, if you specify the base domain `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx:79]:
```suggestion
        - `CONNECT_PROXY_URL`: A URL for the connect proxy subdomain. We recommend you set a first-level subdomain of your `PLATFORM_URL` for your connect proxy. For example, `https://connect.example.com`.
```

File: platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx [hunk 3]
```diff
@@ -68,19 +72,19 @@ This guide assumes that all services will be run in the same container as the re
...
-   1. Uncomment the `connect-proxy` and `connect-server` services.
-   1. Set the following environment variables:
-      - `PLATFORM_URL`: The same value assigned to `TOWER_SERVER_URL`.
-      - `CONNECT_PROXY_URL`: A base domain name for the connect proxy subdomain. For example, if you specify the base domain `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.
-      - `CONNECT_OIDC_CLIENT_REGISTRATION_TOKEN`: The same base64-encoded value set in the `client_token` environment variable.
 
-1. Open `data-studios.env` in an editor and set the following variables:
+    1. Uncomment the `connect-proxy` and `connect-server` services.
+    1. Set the following environment variables:
+        - `PLATFORM_URL`: The same value assigned to `TOWER_SERVER_URL`.
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx:78]:
```suggestion
        - `PLATFORM_URL`: The same value assigned to `TOWER_SERVER_URL`. For example, `https://example.com`.
```

File: platform_versioned_docs/version-24.1.0/enterprise/_templates/k8s/ingress.eks.yml [hunk 4]
```diff
@@ -28,3 +28,14 @@ spec:
             backend:
               serviceName: frontend
               servicePort: 80
+# Uncomment the host section below to enable Data Studios. Replace <CONNECT_DOMAIN> with your Data Studio Connect URL. See [Data Studios configuration](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios) for more information.
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/enterprise/_templates/k8s/ingress.eks.yml:31]:
```suggestion
# Uncomment the host section below to enable Data Studios. Replace <YOUR-TOWER-HOST-NAME> with the base domain of your Platform installation. See [Data Studios configuration](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios) for more information.              
```

File: platform_versioned_docs/version-24.1.0/enterprise/_templates/k8s/ingress.eks.yml [hunk 5]
```diff
@@ -28,3 +28,14 @@ spec:
             backend:
               serviceName: frontend
               servicePort: 80
+# Uncomment the host section below to enable Data Studios. Replace <YOUR-TOWER-HOST-NAME> with the base domain of your Platform installation. See [Data Studios configuration](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios) for more information.              
+#    - host: "*.<CONNECT_DOMAIN>"
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/enterprise/_templates/k8s/ingress.eks.yml:32]:
```suggestion
#    - host: "*.<YOUR-TOWER-HOST-NAME>"
```

File: platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx [hunk 6]
```diff
@@ -122,20 +125,26 @@ This procedure describes how to configure Data Studios for Kubernetes deployment
...
 
     - `CONNECT_REDIS_ADDRESS`: The hostname or IP address of the Redis server configured for Platform.
-    - `CONNECT_PROXY_URL`: A base domain name for the connect proxy subdomain. For example, if you specify the base domain `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.
+    - `CONNECT_PROXY_URL`: A URL for the connect proxy subdomain. We recommend you set a first-level subdomain of your Platform installation domain (`PLATFORM_URL` below) for your connect proxy, to be able to use the same wildcard TLS certificate for all data studio instance URLs and avoid additional domain nesting. For example, `https://connect.example.com`.
     - `PLATFORM_URL`: The base URL for your Platform installation, such as `https://example.com/`.
 
+1. Edit your Platform installation's `ingress.eks.yml` file:
+
+    - Uncomment the `host` section at the bottom of the file.
+    - Replace `<CONNECT_DOMAIN>` with the domain of your `CONNECT_PROXY_URL` value specified in the `proxy.yml` file above. For example, if your connect proxy URL is `https://example.com`, set the `<CONNECT_DOMAIN>` value as `example.com`.
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx:139]:
```suggestion
    - Replace `<YOUR-TOWER-HOST-NAME>` with the base domain of your Platform installation. For example, `example.com`.
```

**Review by endre-seqera — APPROVED** [review]
LGTM!

**Comment by jason-seqera** [comment]
@llewellyn-sl thanks!

**Pull request merged**